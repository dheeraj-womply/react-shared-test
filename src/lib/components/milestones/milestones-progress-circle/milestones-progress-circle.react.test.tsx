import React from 'react';
import renderer from 'react-test-renderer'

import { MilestonesProgressCircle } from './index';
const milestonesConfig = {
    "milestones" : [ {
        "key" : "SAVE_VAULT_CREDENTIALS",
        "description" : "save creds in vault",
        "displayText" : "Save your credentials",
        "urlPage" : "/reputation_settings",
        "urlSubDomain" : '',
        "sortIndex" : 1200,
        "productId" : 9,
        "completedTime" : 0
    }, {
        "key" : "TURN_ON_AUTO_REPLY",
        "description" : "turn on auto reply",
        "displayText" : "Turn on Auto Reply",
        "urlPage" : "/reputation_settings",
        "urlSubDomain" : '',
        "sortIndex" : 1300,
        "productId" : 6,
        "completedTime" : 0
    }],
    "milestoneCount" : 9,
    "completedMilestoneCount" : 1,
    "pollingPages" : [ {
        "milestoneKey" : "VERIFY_EMAIL",
        "urlPage" : "/user-profile",
        "urlSubDomain" : ''
    }, {
        "milestoneKey" : "VERIFY_EMAIL",
        "urlPage" : "/billing",
        "urlSubDomain" : ''
    }]
};

test('DOM elements are OK', () => {
    const component = renderer.create(
        (<MilestonesProgressCircle milestonesData={milestonesConfig}/>)
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
