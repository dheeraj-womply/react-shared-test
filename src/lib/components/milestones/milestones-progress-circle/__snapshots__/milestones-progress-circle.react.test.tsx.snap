// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOM elements are OK 1`] = `
<div
  className="sc-AxjAm qjMPV"
  style={
    Object {
      "background": "linear-gradient(to bottom, #fbda61 11%, #f76b1c 31%)",
    }
  }
>
  <div
    className="sc-AxirZ hrsOry"
  >
    11
    %
  </div>
</div>
`;
